src/averageAges.js

```diff
@@ -20,6 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  // let manList;
+  const manList = !century
```

Comment:
```suggestion
  const manList = century
```
проще читать код, когда отталкиваешься от положительного условия, а не от отрицательного.